src/Command/Social.js

```diff
@@ -3,17 +3,12 @@ const Command = require('../Command');
 module.exports = Command.extend({
     commandName: 'social',
     commandAliases: ['twitter', 'facebook', 'youtube', 'googleplus', 'gplus', 'google+', 'youku', 'playerme', 'instagram', 'tumblr'],
+    config: null,
```

Comment:
This isn't necessary as dependencies are merged with their parent class. Please remove it.